### PR TITLE
fix(migration): make compatible with initial sql file

### PR DESCRIPTION
**Pull request by dsonck92**

### CODE REVIEW

File: migrations/Dockerfile [hunk 1]
```diff
@@ -12,11 +12,16 @@ FROM rust:1.79-alpine AS rust-builder
 
 WORKDIR /build
 
+RUN cargo install cargo-build-dependencies
```
Comment (loboda4450) [inline migrations/Dockerfile:15]:
```suggestion
RUN cargo install cargo-build-dependencies &&\
        apk add --no-cache musl-dev
```

File: migrations/Dockerfile [hunk 2]
```diff
@@ -12,11 +12,16 @@ FROM rust:1.79-alpine AS rust-builder
...
+RUN apk add --no-cache musl-dev
```
Comment (loboda4450) [inline migrations/Dockerfile:16]:
```suggestion

```

**Review by loboda4450 — CHANGES_REQUESTED** [review]
Just as in another PR, less layers is better :D

File: migrations/Dockerfile (hunk 1 above)
Comment (dsonck92) [reply migrations/Dockerfile]: I actually discovered that I could do it without this dependency at all :laughing:

**Review by loboda4450 — APPROVED** [review]
I like that

**Review by loboda4450 — APPROVED** [review]

**Review by bouassaba — DISMISSED** [review]

**Review by bouassaba — DISMISSED** [review]

**Comment by loboda4450** [comment]
@dsonck92 please alter the CI too then

**Comment by dsonck92** [comment]
> @dsonck92 please alter the CI too then

Yeah, just realized. I was a little misguided by the broken `main`, but I had this plan to move it to a "non -s" folder for some time already

**Comment by loboda4450** [comment]
Tbh migrations was lit imo, as idp, api etc. is ONE, but there will be multiple migrations

**Review by loboda4450 — APPROVED** [review]
That's pretty, but I'd personally have this bash script within workflow for clarity, but even without I like the outcome.

**Comment by dsonck92** [comment]
@loboda4450  Yeah, I'm not too pleased with the fact I had to introduce the bash script, I'd rather put it inside the Dockerfile itself. But I had some issues with getting the `case ... in` working. But I realize now that it could have been the lack of the `ARG` that I introduced after the build script.

**Review by loboda4450 — APPROVED** [review]

**Pull request merged**